ncl/ncl_entries/trigonometric_functions.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>I don't think we need to link every single reference of the modules, but I feel like we should link to the docs for <code>math</code>, <code>cmath</code>, and <code>numpy</code> and in the <code>numpy</code>  recommendation section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Do you mean that the references/links for the docs should be moved to the last section? I am currently linking them the first time I reference them and then again at end in Python Resources

Reply:
<p>Oh my bad, sorry, I missed the links on the code formatted text on reviewnb, never mind!</p>